### DISCUSSION TITLE
Appwrite Graphql support

### DISCUSSION THREAD

**Discussion by MoffatMore** [appwrite/appwrite#1280] [Q&A] [answered]
Hello guys, does appwrite support graphql api? or is it still supporting only REST?

**Comment by eldadfux** [comment, marked as answer]
GraphQL is planned and there is already a PR in progress :)
https://github.com/appwrite/appwrite/pull/974

**Reply by MoffatMore** [reply]
Thanks @eldadfux

**Reply by go4cas** [reply]
@eldadfux ... any ETA on the GraphQL feature?

**Reply by eldadfux** [reply]
Very hard to tell right now, but development is in a pretty advanced stage.

**Reply by cto-leaps** [reply]
Any progress / forecast release info? I'm actively looking at switching to Appwrite but I need Graphql support to migrate.

**Reply by eldadfux** [reply]
This is actively being worked on by @abnegate. We still don't have an exact date, but I believe we could share more in 2-4 weeks.

**Reply by badmark** [reply]
Any update?

**Reply by ammarahm-ed** [reply]
Any update on graphql support dropping soon.

**Reply by Caryntjen** [reply]
also wondering @eldadfux

**Reply by cadamsdev** [reply]
Any update?

**Reply by eldadfux** [reply]
This is back at work for a few weeks now, the team has made major progress. We will soon start testing, QA, and benchmarks.

**Reply by anhtuank7c** [reply]
Can I ask for the processing status?

**Reply by eldadfux** [reply]
This feature is planned for Appwrite 1.2, next version is Appwrite 1.0

**Reply by johannesschobel** [reply]
when will appwrite 1.2 arive? is there an ETA? ;) i know, 1.1 is the next release, but just curious ;9

**Reply by christyjacob4** [reply]
@chadalen @johannesschobel @anhtuank7c GraphQL was released as part of Appwrite 1.2!
https://github.com/appwrite/appwrite/releases/tag/1.2.0
